photon-core/src/main/java/org/photonvision/vision/opencv/Contour.java

```diff
@@ -47,6 +47,16 @@ public Contour(MatOfPoint mat) {
         this.mat = mat;
     }
 
+    public Contour(Rect2d box) {
+        // no easy way to convert a Rect2d to Mat, diy it. Order is tl tr br bl
```

Comment:
https://docs.opencv.org/4.x/javadoc/org/opencv/core/MatOfRect2d.html

This relevant?